xml/System.Text.Json.Serialization/BinaryDataJsonConverter.xml

```diff
@@ -109,6 +110,7 @@
         <param name="options">To be added.</param>
         <summary>To be added.</summary>
         <remarks>To be added.</remarks>
+        <inheritdoc />
```

Comment:
@carlossanlop The tool didn't remove any of the "To be added." sections above. Is this expected?

Reply:
They never get removed. But the report will now ignore them since they have inheritdoc.